app/i18n/ja/admin.php

```diff
@@ -160,8 +160,8 @@
 		'_' => 'システム設定',
 		'auto-update-url' => '自動アップグレードするサーバーのURL',
 		'base-url' => array(
-			'_' => 'Base URL',	// TODO
-			'recommendation' => 'Automatic recommendation: <kbd>%s</kbd>',	// TODO
+			'_' => 'ベースURL',
```

Comment:
This item is also corrected to katakana according to other items corrected to katakana.